cpp/ps0004/ps0004_sol.cpp

```diff
@@ -19,3 +23,4 @@ int main() {
 
     return 0;
 }
+// good work 👍 7/10

```

Comment:
- [x] program is in the correct path
- [ ] no syntax errors, bugs or warnings while compilation and execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [ ] main returns non-zero/zero number according to the case
you can check the [problem solution](https://onlinegdb.com/9x6wLTlkK) for more